libs/sdk-core/src/binding.rs

```diff
@@ -143,6 +149,14 @@ pub fn breez_events_stream(s: StreamSink<BreezEvent>) -> Result<()> {
     Ok(())
 }
 
+/// If used, this must be called before `connect`. It can only be called once.
+pub fn breez_node_log_stream(s: StreamSink<LogMessage>) -> Result<()> {
```

Comment:
Yeah there's two log streams here now. I guess that's inevitable. Probably a good solution.